125.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        regex = re.compile('[^a-zA-Z0-9]')
+        letter_str = regex.sub('', s).lower()
```

Comment:
No need to use regular expression. This line can be replaced with list comprehension. 

```python
letters = [char.lower() for char in s if char.isalnum()]